app/lint-baseline.xml

```diff
@@ -1764,6 +1756,39 @@
             column="25"/>
     </issue>
 
+    <issue
+        id="LogConditional"
+        message="The log call Log.v(...) should be conditional: surround with `if (Log.isLoggable(...))` or `if (BuildConfig.DEBUG) { ... }`"
```

Comment:
Oh god, what is even the point of different log levels if we have to do this anyway 😭